doc/conftest.py

```diff
@@ -62,6 +62,13 @@ def setup_compose():
         raise SkipTest("Skipping compose.rst, pandas not installed")
 
 
+def setup_impute():
+    try:
+        import pandas  # noqa
+    except ImportError:
+        raise SkipTest("Skipping impute.rst, pandas not installed")
```

Comment:
I would rather use the following idiom to only skip tests that actually need pandas.

```python
def test_something_with_pandas():
    pd = pytest.importorskip('pandas')
    pd.DataFrame(...)
    ...
```